README.md

```diff
@@ -1,8 +1,7 @@
 # Portfolio-Setup-and-Mobile-first
 
-# Mobile_Project
-> This is our first project that includes more in-depth HTML and CSS. The task involves coding a figma design which mimics our personal project portfolio which we will later be using to present to future employers. The project was added locally (files and linters also set up) and all changes were successful as linters errors were all fixed. The introductory, "my recent works", "about me" and the contact form has been finished for the mobile version.
-
+# Portfolio (Mobile and Desktop)
+>  All changes were successful as linters errors were all fixed. The introductory, "my recent works", "about me" and the contact form page has been finished for the mobile version and desktop version. Media queries were added in order to make the design responsive and animations were also added to certain buttons.
 # Built With
 - html
 - css
```

Comment:
Great job @TayyabKM 🤝

- [ ] [Optional] You may use [markdown syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) to structure your readme file properly. These can help you insert links for your social handles, licenses, and acknowledgments.